README.md

```diff
@@ -1 +1,5 @@
-# ElavonConvergeProcessor
\ No newline at end of file
+# ElavonConvergeProcessor
+
+L3 Config file for working w/ test env: https://s3.amazonaws.com/poynt-configs/L3-Configs-Converge.xml
+
+From MC -> Remote Command -> Load Live Config -> input the above url -> Send
```

Comment:
instructions on how to load L3 config

Reply:
👍 still need `configs/L3-Configs-Elavon.xml` and `configs/Poynt_L3_configs_override.xml`?